docs/commands/pie.md

```diff
@@ -1,82 +1,85 @@
 ## Command pie ##
 
-The `pie` command provides a useful way to set breakpoint to a PIE enabled binary.
-`pie` command then provides what we call "PIE breakpoint". A PIE breakpoint is just
-a virtual breakpoint which will be set to real breakpoint when the process is attaching.
-A PIE breakpoint's address is the offset from binary base address.
-
-Note that you need to use ENTIRE PIE COMMAND SERIES to support PIE breakpoint, especially the
-"attaching" commands provided by `pie` command, like `pie attach`, `pie run`, etc.
-
-Usage is just:
-```
-gef➤ pie <sub_commands>
-```
+The `pie` command provides a useful way to set breakpoint to a PIE enabled
+binary. `pie` command then provides what we call "PIE breakpoints". A PIE
+breakpoint is just a virtual breakpoint which will be turned to a real
```

Comment:
I don't think we need this level of detail the. Maybe just mention that it's handy to resolve to addresses that aren't static.